renku/models/cwl/command_line_tool.py

```diff
@@ -255,7 +292,66 @@ def watch(self, client, no_output=False, outputs=None):
 
         yield tool
 
-        if repo:
+        if repo and self.explicit_outputs:
+            # TODO make sure that output paths are not reserved
+            # TODO check if outputs appear in inputs and warn the user
+            def convert_explicit_outputs_to_cwl_output_parameters():
```

Comment:
1. Move into checks folder
2. Please keep name more Pythonic and put the description into the comment.